packages/nimble-storybook/src/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
What's the idea for nimble and spright living under this src/ tree?
Maybe we should have src/nimble and src/spright?

Reply:
Sure, that's reasonable. Probably some advantage to keeping them separate.